app/gui/src/project-view/components/GraphEditor/widgets/WidgetPort/WidgetPortArrow.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+import { EDGE_ARROW_PATH } from '@/components/GraphEditor/GraphEdges.vue'
+const _props = defineProps<{
+  /** Whether the arrow is enabled and should be displayed. */
+  enabled: boolean
+  /** Whether the arrow should be temporarily hidden (visually). */
+  hide: boolean
```

Comment:
Is this mode to allow it to be hidden but still clickable? I don't see how we are using that, it looks like the only time that both `needsArrow` and `hideArrow` are true is when there is a `mouseEditedEdge`, right?